lib/place_v7.ml

```diff
@@ -488,9 +486,10 @@ let print_html_places_surnames_long conf base link_to_ind
   let a_sort = p_getenv conf.env "a_sort" = Some "on" in
   let f_sort = p_getenv conf.env "f_sort" = Some "on" in
   let up = p_getenv conf.env "up" = Some "on" in
-  let opt = get_opt conf in
-  let list = Array.to_list arry in
-  let list = List.sort (fun (k1, _) (k2, _) -> sort_place_utf8 k1 k2) list in
+  let opt = get_opt conf  (p_getenv conf.env "bump" = Some "on") in
+  let l = Array.to_list arry in
+  let l = List.sort (fun (k1, _) (k2, _) -> sort_place_utf8 k1 k2) l in
+  let l = List.rev l in
```

Comment:
I think if we already have an array `Array.sort` should be faster than `List.sort`, so I would do the conversion to a list after.

Instead of a `List.rev` after the sort you should change the comparison function for it's opposite.